gem/lib/pupperware/spec_helper.rb

```diff
@@ -40,14 +44,14 @@ def run_command(env = ENV.to_h, command)
       end
       Thread.new do
         Thread.current.report_on_exception = false
-        stderr.each { |l| STDOUT.puts l }
+        stderr.each { |l| stderr_string << l and STDERR.puts l }
```

Comment:
I *think* that we dropped things to `STDOUT` so that they'd always show up in spec runs?  But I can't entirely remember... 

Do you have the spec run output from r10k to compare before / after?

Reply:
Original commit at https://github.com/puppetlabs/pupperware/commit/4b29929dfecb3e30b59fab48f7fded909b74eb33

And separation of the streams at https://github.com/puppetlabs/pupperware/commit/56aa6f25bba6b868cccdc725acf236ccf237af51

I didn't write any notes about why we weren't writing to `STDERR`, but could've swore there was a reason - sorry I don't have more to go on here.

Maybe some more clues in the PR https://github.com/puppetlabs/pupperware/pull/40

Reply:
Yeah was wondering about this change too, since it's not a refactor.

I'd rather just write this to `STDOUT` rather than `STDERR` then figure out the history.

Reply:
K reverted this to writing to `STDOUT` and added a comment